### PR TITLE
Update dotnet-35-windows-10.md

### PR DESCRIPTION
**Pull request by mairaw**
CSAT is not terrible on this topic but could be better. Most people commenting are still blocked or don't understand why they need to install the .NET Framework.

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This LGTM.  I've approved this PR and added the hold label.

File: docs/framework/install/dotnet-35-windows-10.md [hunk 1]
```diff
@@ -20,6 +19,10 @@ You may see the following configuration dialog if you try to run an app that req
 
 ![.NET Framework Installation Dialog](./media/dotnet-framework-installation-dialog.jpg)
 
+### Why am I getting this pop-up?
+
+The .NET Framework is a development framework created by Microsoft and used by many companies to develop their apps. Different versions of the .NET Framework already come pre-installed in your Windows machine. When you try to run an app that was developed using version 3.5 of the .NET Framework and you don't have it installed on your machine, you are prompted to do so.
```
Comment (rpetrusha) [inline docs/framework/install/dotnet-35-windows-10.md]: I think this paragraph could use some work. Since this is a user topic, calling something a development framework is unclear and also raises the question of why, as a user, you need a development framework on your machine. I think I'd rephrase this as something like: "The .NET Framework provides an environment for running applications. Applications that run in the .NET Framework target a specific version. If you see this pop-up, you are trying to run an application that requires the .NET Framework Version 3.5, but that version is not installed on your system."

File: docs/framework/install/dotnet-35-windows-10.md [hunk 2]
```diff
@@ -36,4 +39,4 @@ You can enable the .NET Framework 3.5 through the Windows Control Panel. This op
...
 
-If any of the methods discussed in the previous article fail or if you don't have an Internet connection, it's necessary to use your Windows installation media. For more information, see [Deploy .NET Framework 3.5 by using Deployment Image Servicing and Management (DISM)](/windows-hardware/manufacture/desktop/deploy-net-framework-35-by-using-deployment-image-servicing-and-management--dism). If you don't have the installation media, see [Create installation media for Windows](https://support.microsoft.com/help/15088/windows-create-installation-media).
+If you still couldn't resolve your installation issue or you don't have an Internet connection, you can try installing it using your Windows installation media. For more information, see [Deploy .NET Framework 3.5 by using Deployment Image Servicing and Management (DISM)](/windows-hardware/manufacture/desktop/deploy-net-framework-35-by-using-deployment-image-servicing-and-management--dism). If you don't have the installation media, see [Create installation media for Windows](https://support.microsoft.com/help/15088/windows-create-installation-media).
```
Comment (rpetrusha) [inline docs/framework/install/dotnet-35-windows-10.md]: couldn't --> can't

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Left two comments, @mairaw. Your revisions should really help improve customer satisfaction with this topic.

File: docs/framework/install/dotnet-35-windows-10.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/install/dotnet-35-windows-10.md]: I'll rework this. I think it's also important to call out that is developed by Microsoft. Some people think it's a virus.

**Comment by mairaw** [comment]
See if it's better now @rpetrusha

File: docs/framework/install/dotnet-35-windows-10.md [hunk 3]
```diff
@@ -20,6 +19,10 @@ You may see the following configuration dialog if you try to run an app that req
 
 ![.NET Framework Installation Dialog](./media/dotnet-framework-installation-dialog.jpg)
 
+### Why am I getting this pop-up?
+
+The .NET Framework is created by Microsoft and provides an environment for running applications. There are different versions available and your Windows computer already comes with a specific version pre-installed. Many companies develop their apps to run using the .NET Framework and these apps target a specific version. If you see this pop-up, you're trying to run an application that requires the .NET Framework version 3.5, but that version is not installed on your system.
```
Comment (rpetrusha) [inline docs/framework/install/dotnet-35-windows-10.md]: - I think that "and your Windows computer already comes..." can be deleted. It also may not be true (for Windows 7 SP1, e.g.). If you don't delete it, though, there should be a comma after "available".
- comma before "and these apps target..."

**Review by rpetrusha — APPROVED** [review]
This is vastly improved, @mairaw. I've still left a few comments that you can choose to address. Feel free to merge when you're ready.
